packages/tools/pie-icons/bin/build-icons-object.js

```diff
@@ -42,7 +41,7 @@ function buildIconsObject (svgFiles, getSvg) {
  * @param {string} svg
  * @returns {string}
  */
-function getSvgContents (svg) {
+async function getSvgContents (svg) {
```

Comment:
Do you need the async here?

Reply:
Good point 👍  Removed it and updated the JSDoc.